src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -431,7 +431,7 @@ static IEnumerable<(string, string)> ZipWithEmpty(IEnumerable<string> first, IEn
             }
         }
 
-        private static IEnumerable<string> WrapText(string text, int maxWidth)
+        protected static IEnumerable<string> WrapText(string text, int maxWidth)
```

Comment:
This might make more sense as a public utility method so it's available to custom help code without inheriting `HelpBuilder`.

Reply:
See below - I am using most of the methods which were protected before #1483 
- `WriteHeading`
- `GetTwoColumnRow`
- `RenderAsColumns`
If they can be made a public utilities, this may help.  

Also I implemented this helper which might be useful as helper: 
https://github.com/vlada-shubina/templating/blob/83069dbbe891045f6659e4c994fb26e494c11e43/src/Microsoft.TemplateEngine.Cli/Commands/InstantiateHelpBuilder.cs#L569-L595  
In fact, only this method is using `WrapText` above, so if we can expose similar method instead, this would be even better.

If above or similar is moved to `System.CommandLine` exposing `Indent` is not needed at all. 
Besides it, `Indent` is only used in one place when building the custom options layout and it's easy to work around that, and imo it's not a big issue if the indentation will get out of sync.